os/include/tinyara/timer.h

```diff
@@ -129,6 +130,29 @@
  * Public Types
  ****************************************************************************/
 
+/* This is the type of the argument passed to the TCIOC_SETMODE ioctl 
+ * Depending on the argument passed, 
+ * it is decided whether to operate as ALARM or Free running Timer.
+ */
+
+ enum timer_mode_e {
+	MODE_ALARM,
```

Comment:
TIMER_REPEAT, TIMER_FREERUN is more general, I think (alarm means one-shot or repeat....)
Also value shoud consider bit operation, as I told you.

Reply:
i changed that.